docs(router): corrected minor typo

m-ahmadi opened the pull request:
just a typo correction

m-ahmadi commented:
@googlebot I signed it!

atscott commented on packages/router/src/config.ts:
```diff
@@ -420,7 +420,7 @@ export interface Route {
   component?: Type<any>;
   /**
-   * A URL to which to redirect when a the path matches.
+   * A URL to which to redirect when the path matches.
```
```suggestion
   * A URL to redirect to when the path matches.
```
"to which to" appears wrong as well. How about "A URL to redirect to..."?

atscott reviewed (commented):
Hi @m-ahmadi, thanks for you contribution. Can you also update the commit message to conform to the 100 character requirement in the body? I know it's a little tedious on typo corrections, but this character requirement helps us in many other places to prevent PRs that don't describe well enough why something is changing.

atscott commented:
Hi @m-ahmadi, looks like CI failed again. Can you rebase using the steps there:

```
          git fetch upstream master;
          git checkout patch-1;
          git rebase upstream/master;
          git push --force-with-lease;
```

m-ahmadi committed:
docs(router): corrected minor typo

This commit corrects a typo and rephrases the first sentence of the `redirectTo`
description to be more understandable.

m-ahmadi commented:
> Hi @m-ahmadi, looks like CI failed again. Can you rebase using the steps there:
> 
> ```
>           git fetch upstream master;
>           git checkout patch-1;
>           git rebase upstream/master;
>           git push --force-with-lease;
> ```

Hi @atscott, I tried the steps, but nothing seemed to be happening. it said `Current branch patch-1 is up to date.` after the `git rebase ...` command, and also said `Everything up to date.` after the `git push ...` command.

atscott commented:
Hi @m-ahmadi, that's alright. I rebased it and pushed the commit. It may have been a difference with the way your git remotes are named. `upstream` is meant to refer to `git@github.com:angular/angular.git`
